src/wizard/dart/index.md

```diff
@@ -9,7 +9,7 @@ Get the SDK from [pub.dev](https://pub.dev/packages/sentry) by adding the follow
 
 ```yml {filename:pubspec.yaml}
 dependencies:
-  sentry: ^4.0.4
+  sentry: ^{{ packages.version('sentry.dart') }}
```

Comment:
afaik it does not work for wizard, right?

Reply:
@marandaneto works, we have on all wizards